app/templates/companies/new.hbs

```diff
@@ -44,7 +44,7 @@
 
   <div class="row">
     <div class="small-6 small-centered large-12 columns submit-new-company">
-      <a {{action 'submit' target=form}} class="button expand">{{t 'save'}}</a>
+      <a {{action 'saveComanyAndOffer' }} class="button expand">{{t 'save'}}</a>
```

Comment:
@akshaygwd Kindly fix the typo here and in the controller as well. It should be `saveCompanyAndOffer`
Thanks